client/app/components/composites/SearchBar/SearchBar.css

```diff
@@ -184,14 +184,18 @@
   }
 
   &:focus {
-    @media (--large-viewport) {
+    @media (--search-desktop-wide) {
```

Comment:
We need to squeeze the search so that it fits 1200px layout. 
(So, that's 1270px breakpoint where we take 70 pixels away from Search component)